build.gradle

```diff
@@ -40,15 +40,15 @@ project.ext.externalDependency = [
     'commonsLang': 'commons-lang:commons-lang:2.6',
     'ebean': 'io.ebean:ebean:11.33.3',
     'ebeanAgent': 'io.ebean:ebean-agent:11.27.1',
-    'elasticSearchRest': 'org.elasticsearch.client:elasticsearch-rest-high-level-client:5.6.8',
-    'elasticSearchTransport': 'org.elasticsearch.client:transport:5.6.8',
+    'elasticSearchRest': 'org.elasticsearch.client:elasticsearch-rest-high-level-client:7.9.3',
```

Comment:
Is the plan to jump to 7 right away? What about index or data migration (for existing users on 5)?

Reply:
This isn't proposed to go into master but another branch. We can land this outside of master and then discuss how to merge it into master.

My guess is we'll flip the branches, so 5 becomes non-mainline. We'll just have to tick the major version and ask people to be careful with their existing docker setup, if they're pulling our "latest" docker images, which hopefully they aren't.